dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::UserSchoolInfosController < ApplicationController
+  before_action :authenticate_user!
+  before_action :load_user_school_info
+  load_and_authorize_resource
+
+  # PATCH /api/v1/users_school_infos/<id>/update_last_confirmation_date
+  def update_last_confirmation_date
+    @user_school_info.update!(last_confirmation_date: DateTime.now)
+
```

Comment:
Do you need to update `last_seen_school_info_interstitial` here as well?

Reply:
Good question.  Last seen school info interstitial doesn't need to be updated because it is only specific to school info interstitial.